log.graphqls

```diff
@@ -79,4 +79,8 @@ extend type Query {
     # Return true if the current storage implementation supports fuzzy query for logs.
     supportQueryLogsByKeywords: Boolean!
     queryLogs(condition: LogQueryCondition): Logs
+
+    # Test the logs and get the results of the LAL output.
+    # The results are guaranteed to be in JSON format.
+    test(log: [String!]): [String!]
```

Comment:
I think you missed metrics part.

Reply:
> I think you missed metrics part.

All results are in the JSON, including metrics

Reply:
I prefer we don't use this kind of typeless structure. We choose GraphQL as object-oriented, and self-defined. If we are back to String/JSON, we need another doc to write and have to deal with compatibility issue by our own.
Please be clear about all fields.

Reply:
And you only provide log as input, where is the script?